curl-sys/build.rs

```diff
@@ -60,6 +60,9 @@ fn main() {
     let _ = fs::create_dir(&dst.join("build"));
 
     let mut config_opts = Vec::new();
+    if target.len() > 0 {
```

Comment:
I think you can also omit this check because this should always be true I believe.